src/components/Main/Teams/TeamCard.tsx

```diff
@@ -19,12 +19,13 @@ interface TeamCardProps {
 
 export const TeamCard = ({ type, members, draggedItem }: TeamCardProps) => {
   const classes = useStyles();
-  const canDrop: () => boolean = () =>
-    draggedItem.source === 'USERBANK'
+  const canDrop: () => boolean = () => {
+    return draggedItem.source === 'USERBANK'
       ? !members
           .map((member: MembersData) => member.name)
           .includes(draggedItem.member.name)
-      : !(draggedItem.source === type);
+      : draggedItem.source === type;
```

Comment:
Did you remove the `!` on purpose? Because _before_ it said "if it came from the bank, then make sure the member isn't already in the team, otherwise make sure it did _not_ come from the same type". Could this be causing the bug when trying to re-order within a card?

Reply:
Correct. We want it to say "if it came from the bank, then make sure the member isn't already in the team, otherwise make sure it _does_ come from the same type" 

this allows it to reorder

Reply:
So for simplicity, the user can't drag people between teams, and has to drag from the side bar?

Reply:
I merged the fix into the dev branch, so you'll need to pull/resolve merge conflict.